src/main/resources/fuzzing-dictionary.json

```diff
@@ -0,0 +1,14 @@
+{
+  "string": [
+    "null", "", "\\0", 4294967296, 18446744073709551616
+  ],
+  "number": [
+    "null", "", "\\0", -2147483648, 2147483648, 4294967296, 9223372036854775808, -9223372036854775808, 18446744073709551616, 340282346638528860000000000000000000000.000001, -340282346638528860000000000000000000000000001, 179769313486231570000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.000001, -179769313486231570000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000.000001
+  ],
+  "integer": [
+    "null", "", "\\0", -2147483648, 2147483648, 4294967296, 9223372036854775808, -9223372036854775808, 18446744073709551616
+  ],
+  "boolean": [
+    "null", "", "\\0", 4294967296, 18446744073709551616
+  ]
```

Comment:
Create new property "common", to be used by all types, (e.g., including "null", "" and "\\0").